### PR TITLE
[full-ci] [tests-only] Added test for sending share invitation to disabled user

### PR DESCRIPTION
**Pull request by grgprarup**
## Description
Scenarios Added:
- Scenario Outline: send share invitation to disabled user

## Related Issue

- Part of https://github.com/owncloud/ocis/issues/7947

## How Has This Been Tested?

- Locally
- :robot: 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

File: tests/acceptance/features/apiSharingNg/shareInvitations.feature [hunk 1]
```diff
@@ -774,3 +774,90 @@ Feature: Send a sharing invitations
...
+    And user "Alice" has created folder "FolderToShare"
+    And the user "Admin" has disabled user "Brian"
+    When user "Alice" sends the following share invitation using the Graph API:
+      | resourceType    | <resource-type>    |
+      | resource        | <path>             |
+      | space           | Personal           |
+      | sharee          | Brian              |
+      | shareType       | user               |
+      | permissionsRole | <permissions-role> |
+    Then the HTTP status code should be "200"
```
Comment (grgprarup) [inline tests/acceptance/features/apiSharingNg/shareInvitations.feature:790]: Note: We can share with disabled users. https://github.com/owncloud/ocis/issues/7962#issuecomment-1883179994

File: tests/acceptance/features/apiGraph/addUserToGroup.feature [hunk 2]
```diff
@@ -276,7 +276,7 @@ Feature: add users to group
     Given these groups have been created:
       | groupname | comment      |
       | sales     | normal group |
-    And the user "Admin" has disabled user "Alice" using the Graph API
+    And the user "Admin" has disabled user "Alice"
```
Comment (grgprarup) [inline tests/acceptance/features/apiGraph/addUserToGroup.feature:279]: Refactored as we discussed, we are not using texts like `using the Graph API` for a given step.

File: tests/acceptance/features/apiSharingNg/shareInvitations.feature [hunk 3]
```diff
@@ -86,16 +86,16 @@ Feature: Send a sharing invitations
...
-      | Viewer          | file          | /textfile1.txt |
-      | File Editor     | file          | /textfile1.txt |
-      | Co Owner        | file          | /textfile1.txt |
-      | Manager         | file          | /textfile1.txt |
-      | Viewer          | folder        | FolderToShare  |
-      | Editor          | folder        | FolderToShare  |
-      | Co Owner        | folder        | FolderToShare  |
-      | Uploader        | folder        | FolderToShare  |
-      | Manager         | folder        | FolderToShare  |
+      | permissions-role | resource-type | path           |
```
Comment (grgprarup) [inline tests/acceptance/features/apiSharingNg/shareInvitations.feature:89]: Note: kebab-case for example table heading.

**Review by ScharfViktor — APPROVED** [review]

File: tests/acceptance/features/apiSharingNg/shareInvitations.feature [hunk 4]
```diff
@@ -764,13 +764,100 @@ Feature: Send a sharing invitations
...
+      | Co Owner         | file          | /textfile1.txt |
+      | Manager          | file          | /textfile1.txt |
+      | Viewer           | folder        | FolderToShare  |
+      | Editor           | folder        | FolderToShare  |
+      | Co Owner         | folder        | FolderToShare  |
+      | Uploader         | folder        | FolderToShare  |
+      | Manager          | folder        | FolderToShare  |
+
+
+  Scenario Outline: send share invitation to disabled user
```
Comment (saw-jan) [inline tests/acceptance/features/apiSharingNg/shareInvitations.feature:779]: issue here would be helpful for the future
```suggestion
  @issue-
  Scenario Outline: send share invitation to disabled user
```

**Review by PrajwolAmatya — APPROVED** [review]

**Pull request merged**